Dockerfile

```
@@ -90,8 +90,8 @@ RUN tar xf node-v${NODEJS_VERSION}-linux-x64.tar.xz && \
 
 # Add renovate user and switch to it
 RUN useradd -lms /bin/bash -u 1001 -g 0 renovate
-RUN chmod -R 6775 /home/renovate && chown :0 /home/renovate
-RUN mkdir -p /home/renovate/.cache && chown :0 /home/renovate/.cache && chmod 6775 /home/renovate/.cache
+RUN mkdir -p /home/renovate/.cache /home/renovate/.local /home/renovate/.local/state/pdm
+RUN chmod -R 6775 /home/renovate && chown -R 1001:0 /home/renovate && chmod -R 6775 /home/renovate
```

```suggestion
RUN chown -R 1001:0 /home/renovate && chmod -R 2775 /home/renovate
```
There is redundant chmod command, and setuid won't affect dirs.